libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.service.ts

```diff
@@ -59,6 +60,24 @@ export class NationalRegistryXRoadService {
     }))
   }
 
+  async getNationalRegistryChildGuardianship(
+    user: User,
+    nationalId: string,
+  ): Promise<NationalRegistryChildGuardianship | undefined> {
+    const residenceParent = await this.nationalRegistryApiWithAuth(user)
+      .einstaklingarGetBusetuForeldri({ barn: nationalId })
+      .catch(this.handle400)
+    const domicileParent = await this.nationalRegistryApiWithAuth(user)
+      .einstaklingarGetLogheimilisForeldri({ barn: nationalId })
+      .catch(this.handle400)
```

Comment:
Does `einstaklingarGetBusetuForeldri` and `einstaklingarGetLogheimilisForeldri` endpoints from the 3rd party api handle that the nationalID of the child is linked to the user from `nationalRegistryApiWithAuth`?
In other words, can you fetch any child's nationalId?

Reply:
Ah, good point. I'll force a check

Reply:
Very good point, i'll be in touch with Þjóðskrá. The assumption is that the endpoint handles it, but i'm not sure.

Reply:
I'll be relegating this to a draft since the robustness of the api endpoints are currently in question, pick it up later when they're ready